# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kismet/check_snprintf.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN kismet-2004-04-R1/configure.in kismet-2004-04-R1-new/configure.in
--- kismet-2004-04-R1/configure.in	2004-03-17 22:59:34.000000000 +0100
+++ kismet-2004-04-R1-new/configure.in	2004-05-04 13:03:49.265078384 +0200
@@ -164,6 +164,13 @@
 AC_FUNC_STAT
 AC_CHECK_FUNCS([gettimeofday memset select socket strcasecmp strftime strstr])
 
+needsnprintf=no
+AC_CHECK_FUNCS(vsnprintf snprintf,,
+        [needsnprintf=yes])
+if test $needsnprintf = yes; then
+        AC_LIBOBJ(snprintf)
+fi
+
 # Do we have getopt_long natively?
 AC_MSG_CHECKING([for system-level getopt_long()])
 AC_LINK_IFELSE([AC_LANG_PROGRAM([[
